internal/core/metadata/rest_devicereport.go

```diff
@@ -261,31 +283,39 @@ func restDeleteReportByName(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err = deleteDeviceReport(dr, w); err != nil {
-		LoggingClient.Error(err.Error())
+	if err = deleteDeviceReport(dr, w, loggingClient); err != nil {
+		loggingClient.Error(err.Error())
 		return
 	}
 	w.Header().Set(clients.ContentType, clients.ContentTypeJSON)
 	w.WriteHeader(http.StatusOK)
 	w.Write([]byte("true"))
 }
 
-func deleteDeviceReport(dr models.DeviceReport, w http.ResponseWriter) error {
+func deleteDeviceReport(
+	dr models.DeviceReport,
+	w http.ResponseWriter,
+	loggingClient logger.LoggingClient) error {
+
 	if err := dbClient.DeleteDeviceReportById(dr.Id); err != nil {
 		httpErrorHandler.Handle(w, err, errorconcept.Common.DeleteError)
 		return err
 	}
 
 	// Notify Associates
-	if err := notifyDeviceReportAssociates(dr, http.MethodDelete); err != nil {
+	if err := notifyDeviceReportAssociates(dr, http.MethodDelete, loggingClient); err != nil {
 		return err
 	}
 
 	return nil
 }
 
 // Notify the associated device services to the device report
-func notifyDeviceReportAssociates(dr models.DeviceReport, action string) error {
+func notifyDeviceReportAssociates(
+	dr models.DeviceReport,
+	action string,
+	loggingClient logger.LoggingClient) error {
```

Comment:
@michaelestrin This is extremely minor, but what do we think about renaming loggingClient to lc now that the scope is no longer global?

Reply:
@jdharms and I discussed off-line earlier today.  Decision reached to leave implementation here (and elsewhere) as is and handle parameter naming conversion in subsequent PR(s).